lib/Ast_to_Mir.ml

```diff
@@ -103,7 +103,7 @@ let trans_printable (p : Ast.typed_expression Ast.printable) =
   | Ast.PString s -> Lit (Str, unquote s)
   | Ast.PExpr e -> trans_expr e
 
-let rec trans_stmt {Ast.stmt_typed; stmt_typed_loc; _} =
+let rec trans_stmt adt {Ast.stmt_typed; stmt_typed_loc; _} =
   let or_skip = Option.value ~default:Skip in
```

Comment:
might be slightly prettier to define here `let recurse = trans_stmt adt in`

Reply:
or, and maybe this is crazy, `let trans_stmt = trans_stmt adt in` 🐙  but idk if that's more of less confusing...

Reply:
went with the first

Reply:
@seantalts , are you happy with the fix?